lib/Doctrine/ORM/Mapping/NamingStrategy.php

```diff
@@ -61,7 +61,7 @@ public function referenceColumnName();
      *
      * @return string A join column name.
      */
-    public function joinColumnName($propertyName/*, $className = null */);
+    public function joinColumnName($propertyName/*, string $className */);
```

Comment:
Not that it matters in a comment, but 3.0 has a nullable type.

Reply:
Not after #9755 it won't :wink: